ui/apps/platform/src/Containers/Vulnerabilities/searchOptions.ts

```diff
@@ -53,6 +54,12 @@ export const NODE_CVE_SEARCH_OPTION = {
     category: 'NODE_VULNERABILITIES',
 } as const;
 
+export const SNOOZED_NODE_CVE_SEARCH_OPTION = {
+    label: 'CVE Snoozed',
+    value: 'CVE Snoozed',
+    category: 'NODE_VULNERABILITIES',
+} as const;
+
```

Comment:
Like the `parseWorkloadQuerySearchFilter` helper, making changes here is likely throwaway work. In this case though, the change is minimal and straightforward so I added the option here for the sake of PR demonstration.